From 8f4178bc86c891d45b2491ba66a169a0ccf137e9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20La=C3=9F?= <bevan@bi-co.net>
Date: Sun, 9 May 2021 17:48:29 +0200
Subject: [PATCH 1/4] Remove -liio from linker flags when using CMAKE

When requesting support for PLUTO via CMAKE, it configures
pluto-handler-2, which does not link against libiio but instead uses
dlopen() to dynamically load the library at runtime.

Remove the flag to make qt-dab compile with pluto-handler-2 enabled on
systems without libiio.
---
 dab-maxi/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dab-maxi/CMakeLists.txt b/dab-maxi/CMakeLists.txt
index eb8f458..10b8b6f 100755
--- a/dab-maxi/CMakeLists.txt
+++ b/dab-maxi/CMakeLists.txt
@@ -691,7 +691,6 @@ endif ()
 	        ./qt-devices/pluto-handler-2/pluto-handler.cpp
 	   )
 
-	   list(APPEND extraLibs -liio)
 	   add_definitions (-DHAVE_PLUTO)
 	endif (PLUTO)
 #
-- 
2.31.1

